unstructured/partition/docx.py

```diff
@@ -470,11 +516,10 @@ def page_is_odd() -> bool:
         # -- predict when two page breaks will be needed and emit one of them. The second will be
         # -- emitted by the rendered page-break to follow.
 
-        if start_type == WD_SECTION_START.EVEN_PAGE:
+        if start_type == WD_SECTION_START.EVEN_PAGE and not page_is_odd():
             # -- on an even page we need two total, add one to supplement the rendered page break
             # -- to follow. There is no "first-document-page" special case because 1 is odd.
-            if not page_is_odd():
-                yield from self._increment_page_number()
+            yield from self._increment_page_number()
```

Comment:
I'd be inclined to leave this as it was. If the type-checker or linter complains we should place an exception comment. The reason being that a start type is either even or odd and can't be both, so the top-level logic is just partitioning these two cases. The following branch (testing whether start-type is odd) should not be executed if start-type is even.

It's a small thing, bit I did it that way on purpose for that reason, even though this new logic seems simpler to the static checker.

Reply:
Sounds good. Yah, that was changed because of the linter. Will add an exception